classes/cargo.bbclass

```diff
@@ -23,7 +23,8 @@ B = "${S}"
 export RUST_BACKTRACE = "1"
 
 RUSTFLAGS ??= ""
-CARGO_BUILD_FLAGS = "-v --target ${HOST_SYS} --release"
+BUILD_MODE = "${@['--release', ''][d.getVar('DEBUG_BUILD') == '1']}"
```

Comment:
Probably want to adjust `CARGO_TARGET_SUBDIR` below as well so that binaries are automatically picked up by `cargo_do_install`.

Reply:
Good catch! I've updated the pull request to adjust CARGO_TARGET_SUBDIR.